addons/block_code/ui/main_panel.gd

```diff
@@ -129,6 +141,7 @@ func save_script():
 	var generated_script = _block_canvas.generate_script_from_current_window(block_script.script_inherits)
 	block_script.block_trees = block_trees
 	block_script.generated_script = generated_script
+	block_script.version = Constants.CURRENT_DATA_VERSION
```

Comment:
Can we arbitrarily update the version in the saved script if it hasn't been migrated? This is probably fine for now but eventually I'd expect `save_script` to bail if `_current_block_code_node.version != Constants.CURRENT_DATA_VERSION`. Maybe `generate_script_from_current_window` is the one doing the heavy lifting there and should return `null` if it can't understand the blocks on screen.

Reply:
Yes, this is TBD. I guess we'll try a migration and if that fails the blocks shouldn't be editable at all, can't be saved.